mne/preprocessing/tests/test_ica.py

```diff
@@ -252,7 +252,8 @@ def test_ica_core(method):
             ica.fit(epochs[0:0])
 
         # test decomposition
-        with pytest.warns(UserWarning, match='did not converge'):
+        with pytest.warns(UserWarning, match=('An instance of Raw, ' +
```

Comment:
no need for `+` at the end of the line it happens automatically with `str`

Reply:
I don't think you got this yet, unresolving

Also I'm not sure why you need this change at all for this changeset?